build.sbt

```diff
@@ -21,16 +20,18 @@ resolvers ++= Seq(
 
 libraryDependencies ++= Seq(
   "org.webjars" % "swagger-ui" % "4.1.3",
-  "org.broadinstitute.dsde" %%  "rawls-model" % rawlsModelV
-    exclude("bio.terra", "workspace-manager-client")
-    exclude("com.google.api-client", "google-api-client"),
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
+  "com.typesafe.akka" %% "akka-http-spray-json" % "10.2.9",
   "org.broadinstitute.dsde.workbench" %%  "workbench-google" % workbenchGoogleV
     exclude("com.typesafe.akka", "akka-protobuf-v3_2.13")
     exclude("com.google.protobuf", "protobuf-java")
     exclude("org.bouncycastle", "bcprov-jdk15on")
     exclude("org.bouncycastle", "bcprov-ext-jdk15on")
     exclude("org.bouncycastle", "bcutil-jdk15on")
     exclude("org.bouncycastle", "bcpkix-jdk15on"),
+  "org.broadinstitute.dsde.workbench" %% "workbench-notifications" % "0.3-25c2179"
```

Comment:
Thurloe was getting its notification models from rawls-model, but that was removed in https://github.com/broadinstitute/rawls/pull/1697